### ISSUE TITLE
Not pulling all subregions for country India

### ISSUE THREAD

**Issue by Bongs** [carmen-ruby/carmen#218]
I tried to pull all regions for country India on rails console. It didn't pull the region added in "iso_data/overlay/world/in.yml".

Ran following:

```
2.4.0 :011 > cc = Carmen::Country.coded('IN')
 => <#Carmen::Country name="India">
2.4.0 :012 > cc.subregions
 => [<#Carmen::Region name="Andaman and Nicobar Islands" type="union territory">, <#Carmen::Region name="Andhra Pradesh" type="state">, <#Carmen::Region name="Arunachal Pradesh" type="state">, <#Carmen::Region name="Assam" type="state">, <#Carmen::Region name="Bihar" type="state">, <#Carmen::Region name="Chandigarh" type="union territory">, <#Carmen::Region name="Chhattisgarh" type="state">, <#Carmen::Region name="Damen and Diu" type="union territory">, <#Carmen::Region name="Delhi" type="union territory">, <#Carmen::Region name="Dadra and Nagar Haveli" type="union territory">, <#Carmen::Region name="Goa" type="state">, <#Carmen::Region name="Gujarat" type="state">, <#Carmen::Region name="Himachal Pradesh" type="state">, <#Carmen::Region name="Haryana" type="state">, <#Carmen::Region name="Jharkhand" type="state">, <#Carmen::Region name="Jammu and Kashmir" type="state">, <#Carmen::Region name="Karnataka" type="state">, <#Carmen::Region name="Kerala" type="state">, <#Carmen::Region name="Lakshadweep" type="union territory">, <#Carmen::Region name="Maharashtra" type="state">, <#Carmen::Region name="Meghalaya" type="state">, <#Carmen::Region name="Manipur" type="state">, <#Carmen::Region name="Madhya Pradesh" type="state">, <#Carmen::Region name="Mizoram" type="state">, <#Carmen::Region name="Nagaland" type="state">, <#Carmen::Region name="Orissa" type="state">, <#Carmen::Region name="Punjab" type="state">, <#Carmen::Region name="Puducherry" type="union territory">, <#Carmen::Region name="Rajasthan" type="state">, <#Carmen::Region name="Sikkim" type="state">, <#Carmen::Region name="Tamil Nadu" type="state">, <#Carmen::Region name="Tripura" type="state">, <#Carmen::Region name="Uttar Pradesh" type="state">, <#Carmen::Region name="Uttarakhand" type="state">, <#Carmen::Region name="West Bengal" type="state">]
2.4.0 :013 > cc.subregions.count
 => 35
```

It doesn't list the region "Telangana".

**Comment by dainmiller** [comment]
It looks like here https://github.com/jim/carmen/pull/219/files you modified a code instead of creating a new one? But in this issue you seem to be referring to the fact that all regions aren't included. Am I just confusing something?

**Comment by Bongs** [comment]
That pull request is just to correct the code (may be not related with the issue). In my app, I fixed it by creating similar folder structure used by carmen, e.g.:
```
my_app/lib/carmen/iso_data/overlay/world/in.yml
---
- code: TG
  type: state

my_app/lib/carmen/locale/overlay/en/world/in.yml
---
en:
  world:
    in:
      tg:
        name: Telangana
```

And had to create initializer in ```my_app/config/initializers/carmen.rb``` with following code:
```
require 'carmen'
Carmen.append_data_path Rails.root.join('lib', 'carmen', 'iso_data', 'overlay')
Carmen.i18n_backend.append_locale_path Rails.root.join('lib', 'carmen', 'locale', 'overlay')
```

Not sure why I had to do that. I thought the gem takes care of including the files in overlay folder and merging the regions.

**Comment by swcraig** [comment]
Thanks for reporting. I couldn't recreate this. Master had Telagana even before I merged #219.
